### DISCUSSION TITLE
How to Uninstall/remove Rehike

### DISCUSSION THREAD

**Discussion by nailuj9002** [Rehike/Rehike#538] [Q&A] [answered]
When i use rehike for some reason youtube takes longer to buffer videos so i've decided to remove rehike but i cant find any information about Uninstalling Rehike in the FAQ or reddit (ik reddit sucks) so i need help

**Comment by ephemeralViolette** [comment]
Follow the installation instructions in reverse. Namely, remove the hosts file override and you should just get non-Rehike YouTube back.

**Comment by RealPortalPlayer** [comment, marked as answer]
Just remove the line from your hosts file, or remove your proxy. You can also disable Rehike via a button in Rehike settings. It's the "Disable Rehike" button [here.](https://www.youtube.com/rehike/config)

**Reply by kawapure** [reply]
Note that the "disable Rehike" feature still causes network traffic to be proxied through the Rehike server. It works almost perfectly, but certain bugs affecting the Rehike server can bleed through.

A recent example that comes to mind is that Studio requests to change images visible on the user's profile were being rejected because of a PHP warning muddying the response data which the Studio JS read. This was only fixed very recently (in 9c95caffa3311318c407ac81171d1ad7e1d65695).
